universal-application-tool-0.0.1/app/views/admin/programs/ProgramAdministratorProgramListView.java

```diff
@@ -90,11 +90,7 @@ public Tag renderProgramListItem(
 
     Tag midContent =
         div(programDescriptionText)
-            .withClasses(
-                Styles.TEXT_GRAY_700,
-                Styles.TEXT_BASE,
-                Styles.MB_8,
-                "line-clamp-3" /* TODO: Add tailwind plugin for line clamping. */);
+            .withClasses(Styles.TEXT_GRAY_700, Styles.TEXT_BASE, Styles.MB_8, Styles.LINE_CLAMP_3);
```

Comment:
What is the affordance for expanding the description to see the entire thing? I explained some thoughts in #1196 and #1180 .

Reply:
Replied on #1196.